server/routes/api.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express')
+const axios = require('axios')
+const City = require('../../city')
+const moment = require('moment')
+
+const route = express()
+const key = '209e5b6a8d75041260674c4076a7f389'
```

Comment:
you should read about .env file and how to use it - usually, we don't put our API keys in the JS files, but in the .env file